scripts/grab_operator_yamls.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This scripts obtains proto files used by the Istio project from upstream
+# locations. The proto file's repository is pinned. The pinned protos
+# are then copied into a directory called "protos".
+
+# Find the output directory
+scriptpath="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+rootdir=$(dirname "${scriptpath}")
+output_dir="${rootdir}/static"
+
+# Temporary directories securely created
+tempdir_operator="$(mktemp -d)"
+
+# Upstream GIT tags or branches used for protobufs by repo
+operator_tag="master"
+
+# Great care should be taken when modifying the ordering of this list. This
+# script cats these files together in order with a yaml separator.
+
+operator_manifest_files=( "deploy/namespace.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_crd.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml" "deploy/service_account.yaml" "deploy/clusterrole.yaml" "deploy/clusterrole_binding.yaml" "deploy/service.yaml" "deploy/operator.yaml" )
+
+# Add profiles here to have them automatically added to the website.
+# It is important to also document these files, which is not done automatically.
+
+operator_profiles=( "default" "demo" "demo-auth" "sds" "minimal" )
+
+rm -f "${output_dir}"/operator.yaml
+touch "${output_dir}"/operator.yaml
+echo "operator/*"
+pushd "${tempdir_operator}" >/dev/null || exit
+git clone -q --single-branch --branch master https://github.com/istio/operator.git
```

Comment:
Do you want to be using $operator_tag here instead of 'master'?

Reply:
done.